elastic/apm-server
# backport: enable auto-merge on backport PRs

## Motivation/summary

Enable --autoMerge on backport PRs, to remove one more manual step from the process.

## How to test these changes

Run backport ([5.6.6](https://github.com/sqren/backport/releases/tag/v5.6.6)+), see that auto-merge is enabled.
